test/integration/src/main/java/io/pravega/test/integration/utils/TestUtils.java

```diff
@@ -0,0 +1,115 @@
+            if (!isStreamCreated) {
+                throw new RuntimeException("Failed to create stream: " + s);
+            }
+        });
+    }
+
+    @SneakyThrows
+    public static void writeDataToStream(String scope1, String stream1, String message1, ClientConfig writerClientConfig) {
+        @Cleanup final EventStreamClientFactory writerClientFactory = EventStreamClientFactory.withScope(scope1,
+                writerClientConfig);
+
+        @Cleanup final EventStreamWriter<String> writer = writerClientFactory.createEventWriter(stream1,
+                new JavaSerializer<String>(),
+                EventWriterConfig.builder().build());
+        writer.writeEvent(message1).get();
+        log.info("Wrote a message to the stream {}/{}", scope1, stream1);
+    }
+
+    public static String readAMessageFromStream(String scopeName, String streamName, ClientConfig readerClientConfig,
+                                                 String readerGroupName) {
+        @Cleanup
+        EventStreamClientFactory readerClientFactory = EventStreamClientFactory.withScope(scopeName, readerClientConfig);
+        log.debug("Created the readerClientFactory");
+
+        ReaderGroupConfig readerGroupConfig = ReaderGroupConfig.builder()
+                .stream(Stream.of(scopeName, streamName))
+                .disableAutomaticCheckpoints()
+                .build();
+        @Cleanup
+        ReaderGroupManager readerGroupManager = ReaderGroupManager.withScope(scopeName, readerClientConfig);
+        readerGroupManager.createReaderGroup(readerGroupName, readerGroupConfig);
+        log.debug("Created reader group with name {}", readerGroupName);
+
+        @Cleanup
+        EventStreamReader<String> reader = readerClientFactory.createReader(
+                "readerId", readerGroupName,
+                new JavaSerializer<String>(), ReaderConfig.builder().initialAllocationDelay(0).build());
+        log.debug("Created an event reader");
+
+        // Keeping the read timeout large so that there is ample time for reading the event even in
+        // case of abnormal delays in test environments.
+        String readMessage = reader.readNextEvent(10000).getEvent();
+        log.info("Done reading event [{}]", readMessage);
+        return readMessage;
+    }
+
+    public static List<PasswordAuthHandlerInput.Entry> preparePasswordInputFileEntries(
+            Map<String, String> entries, String password) {
+        StrongPasswordProcessor passwordProcessor = StrongPasswordProcessor.builder().build();
+        try {
+            String encryptedPassword = passwordProcessor.encryptPassword(password);
+            List<PasswordAuthHandlerInput.Entry> result = new ArrayList<>();
+            entries.forEach((k, v) -> result.add(PasswordAuthHandlerInput.Entry.of(k, encryptedPassword, v)));
+            return result;
+        } catch (NoSuchAlgorithmException | InvalidKeySpecException e) {
```

Comment:
You already used `SneakyThrows` in this class. You might as well use it here too.

Reply:
Done.